NorthstarDLL/core/hooks.cpp

```diff
@@ -7,7 +7,7 @@
 #include <fstream>
 #include <sstream>
 #include <filesystem>
-#include <Psapi.h>
+#include <psapi.h>
```

Comment:
Isn't `Psapi` the correct casing for this one?

Reply:
according to Microsoft `psapi.h` appears correct
https://learn.microsoft.com/en-us/windows/win32/api/psapi/

Reply:
Given that online documentation appears rarely correct I've gone to verfiy this and
![image](https://github.com/R2Northstar/NorthstarLauncher/assets/15076013/87680ea2-5850-4351-ba09-64ee8bb4529d)

Since the header can be used on case sensitive system I'll also revert this.